Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -3985,8 +3985,10 @@ script:
       required: true
       predefined:
       - Audit
+      - Block
```

Comment:
no need to add this to the readme?

Reply:
doesn't seem like those values appear in the readme at all